src/SFA.DAS.LevyTransferMatching.Domain/Extensions/PledgeStatusExtenstions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using SFA.DAS.LevyTransferMatching.Domain.Types;
+
+namespace SFA.DAS.LevyTransferMatching.Domain.Extensions
+{
+    public static class PledgeStatusExtenstions
+    {
+        public static string GetLabelForPledgeStatus(this PledgeStatus status)
+        {
+            switch (status)
+            {
+                case PledgeStatus.Active: return "ACTIVE";
+                case PledgeStatus.Closed: return "CLOSED";
+                default:
+                    return string.Empty;
+            }
+        }
+
+        public static string GetCssClassForPledgeStatus(this PledgeStatus status)
```

Comment:
See comment above on name

Reply:
@chrisfoster76 Yes sure, these changes are now done, I have renamed it to GetCssClass